src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DeleteIT.java

```diff
@@ -77,12 +68,76 @@ public void deleteWithConditionTest() throws IOException, InterruptedException {
         // To prevent flakiness, the minimum value of 2000 msec works fine.
         Thread.sleep(2000);
 
-        selectQuery = String.format(
-            Locale.ROOT,
-            "SELECT * FROM %s/phrase",
+        selectQuery = StringUtils.format("SELECT * FROM %s/phrase", TestsConstants.TEST_INDEX_PHRASE);
+
+        response = executeRequest(makeRequest(selectQuery));
+        assertThat(getTotalHits(response), equalTo(5));
+    }
+
+    @Test
+    public void deleteAllWithJdbcFormat() throws IOException, InterruptedException {
+        String selectQuery = StringUtils.format("SELECT * FROM %s/account", TestsConstants.TEST_INDEX_ACCOUNT);
+        JSONObject response = executeRequest(makeRequest(selectQuery));
+        int totalHits = getTotalHits(response);
+
+        String deleteQuery = StringUtils.format("DELETE FROM %s/account", TestsConstants.TEST_INDEX_ACCOUNT);
+
+        response = new JSONObject(executeQuery(deleteQuery, "jdbc"));
+        System.out.println(response);
+        assertThat(response.query("/schema/0/name"), equalTo("deleted_rows"));
+        assertThat(response.query("/schema/0/type"), equalTo("long"));
+        assertThat(response.query("/datarows/0/0"), equalTo(totalHits));
+        assertThat(response.query("/total"), equalTo(1));
+        assertThat(response.query("/status"), equalTo(200));
+        assertThat(response.query("/size"), equalTo(1));
+
+        // The documents are not deleted immediately, causing the next search call to return all results.
+        // To prevent flakiness, the minimum value of 2000 msec works fine.
+        Thread.sleep(2000);
+
+        response = executeRequest(makeRequest(selectQuery));
+        assertThat(getTotalHits(response), equalTo(0));
+
+        // Multiple invocation of delete query should return deleted == 0
+        response = new JSONObject(executeQuery(deleteQuery, "jdbc"));
+        assertThat(response.query("/datarows/0/0"), equalTo(0));
+    }
+
+    @Test
```

Comment:
Perhaps not necessary- but I think it could be beneficial to add 1-2 more variety of queries to test with

Reply:
Can you provide examples of what you kind of variety you are expecting.